workers/loc.api/sync/balance.history/index.js

```diff
@@ -261,15 +267,13 @@ class BalanceHistory {
         return accum + balance
       }
 
-      const prise = {
-        ...this._getCandlesClosePrice(
-          candles,
-          mts,
-          timeframe,
-          `t${symb}USD`,
-          currenciesSynonymous
-        )
-      }
+      const prise = this._getCandlesClosePrice(
```

Comment:
Please remember to fix this and where price is written wrongly

Reply:
ok sure, thanks for the pointing

Reply:
also, the typo is fixed in the next PR where the corresponding file is changed
[#137-commit346dfdf](https://github.com/bitfinexcom/bfx-reports-framework/pull/137/commits/346dfdf6bf4e37014baa3d3c1bf8b59a4373aac1)